leaderboardscoring/repository/dbconfig.yml

```
@@ -0,0 +1,10 @@
+postgres_db:
+  driver: postgres
+  host: leaderboardscoring-db
+  port: 5432
+  user: leaderboardscoring_admin
+  password: password123
+  db_name: leaderboardscoring_db
+  ssl_mode: disable
+
+path_of_migration: ./leaderboardscoring/repository/migrations
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)